glean-core/metrics.yaml

```diff
@@ -244,3 +244,18 @@ glean.error:
       - all_pings
     no_lint:
       - COMMON_PREFIX
+
+glean.deletion_request:
+  optout:
+    type: boolean
+    description:
+      Signals that the user opted out of sending technical and interaction data.
```

Comment:
Ah, we're not going for the "send if empty flag" for this. That's fine in the short term, provided that: (a) we file bout about adding that to `pings.yaml` to support this and other cases (b) document in this metric that this is really useless and it's only really needed to make glean send the ping out. Let's also reference the bug number for adding that feature, please.

Reply:
That would also be blocked on the nice Rust API, right? If so, any chance you could mention this as well?

Reply:
to answer the first: yes, I think going for the flag is the better option. What do you think about me implementing that first and landing it, before we land the deletion ping here?

Reply:
I have no strong opinion on this, I'd leave it to your judgment :-) If you want to play safe, given that nobody is going to rely on that data point, just land the current version without blocking on that flag, so that we have enough time to validate this once it hits Fenix.